.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will test a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Test with Maven
+      run: mvn test --file walkthroughs/week-5-tdd/project/pom.xml
```

Comment:
nit: you can add `-B` and `-ntp` to make the build logs a bit cleaner

Reply:
-ntp is not a flag I could find